### PR TITLE
delete getWells and change illuminate requirements

### PR DESCRIPTION
**Pull request by simbig**
- [ ] Added automated tests
- [ ] Documented for all relevant versions
- [x] Updated the changelog

**Changes**

- Add compatibility to Illuminate\Support `^6|^7|^8|^9`

**Breaking changes**

- Removed method `getWells()` from class `MicroPlate` - use `wells` property of `MicroPlate` class

### CODE REVIEW

File: tests/Unit/MicroPlateTest.php [hunk 1]
```diff
@@ -25,8 +25,8 @@ public function testCanAddAndRetriedWellBasedOnCoordinateSystem(): void
         $wellContent2 = 'bar';
         $microPlate->addWell($microPlateCoordinate2, $wellContent2);
 
-        self::assertEquals([$microPlateCoordinate1, $wellContent1], ($microPlate->getWells())[0]);
-        self::assertEquals([$microPlateCoordinate2, $wellContent2], ($microPlate->getWells())[1]);
+        self::assertEquals([$microPlateCoordinate1, $wellContent1], ($microPlate->wells)[0]);
```
Comment (spawnia) [inline tests/Unit/MicroPlateTest.php]:
```suggestion
        self::assertEquals([$microPlateCoordinate1, $wellContent1], $microPlate->wells[0]);
```

File: tests/Unit/MicroPlateTest.php [hunk 2]
```diff
@@ -25,8 +25,8 @@ public function testCanAddAndRetriedWellBasedOnCoordinateSystem(): void
...
+        self::assertEquals([$microPlateCoordinate2, $wellContent2], ($microPlate->wells)[1]);
```
Comment (spawnia) [inline tests/Unit/MicroPlateTest.php]:
```suggestion
        self::assertEquals([$microPlateCoordinate2, $wellContent2], $microPlate->wells[1]);
```

File: composer.json [hunk 3]
```diff
@@ -12,7 +12,7 @@
   ],
   "require": {
     "php": "^7.4 || ^8",
-    "laravel/framework": "^7.30",
+    "illuminate/support": "^6 || ^7 || ^8 || ^9",
```
Comment (spawnia) [inline composer.json]: Test-Matrix should be expanded to cover those actually work

File: .github/workflows/continuous-integration.yml [hunk 4]
```diff
@@ -179,6 +185,9 @@ jobs:
       - name: "Run unit tests with phpunit/phpunit"
         run: vendor/bin/phpunit
 
+      - name: "Check PSR-12 Codestyle"
```
Comment (spawnia) [inline .github/workflows/continuous-integration.yml]: Why the random addition of this tool after PHPUnit? Why should it run in the matrix, is PSR12 different for each PHP version and/or Laravel version?

**Review by spawnia — CHANGES_REQUESTED** [review]

File: .github/workflows/continuous-integration.yml [hunk 5]
```diff
@@ -153,6 +153,15 @@ jobs:
           - locked
```
Comment (spawnia) [inline .github/workflows/continuous-integration.yml]: Makes little sense now that we dynamically change major versions of our dependencies

File: .github/workflows/continuous-integration.yml (hunk 4 above)
Comment (simbig) [reply .github/workflows/continuous-integration.yml]: it was just a test, deleted it anyway

File: .github/workflows/continuous-integration.yml (hunk 5 above)
Comment (simbig) [reply .github/workflows/continuous-integration.yml]: so what needs to be done? Just delete "locked"?

Comment (spawnia) [reply .github/workflows/continuous-integration.yml]: jup

**Review by spawnia — APPROVED** [review]

**Pull request merged**